services/state/addressbook/node.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package com.hedera.hapi.node.state.addressbook;
+package com.hedera.hapi.node.addressbook;
```

Comment:
This should have the state sub-package to match what we're doing with other protobufs.

Reply:
Updated.